### PR TITLE
Update regex

### PR DESCRIPTION
**Pull request by XyntaxCS**
Update the regex for autocompletion tot support indentations and bullet points. Should be tested for both functionality and performance. Currently I do not run into any trouble.

### CODE REVIEW

File: editor/src/kroqed-editor/menubar/menubar.ts [hunk 1]
```diff
@@ -139,16 +142,16 @@ function createDefaultMenu(schema: Schema, outerView: EditorView, filef: any, os
     // Create the list of menu entries.
     let items: MenuEntry[] = [
         // Insert Coq command
-        createMenuItem("Coq↓", `Insert new coq cell underneath (${cmdOrCtrl}-q)`, cmdInsertCoq(schema, filef, InsertionPlace.Underneath), false),
-        createMenuItem("Coq↑", `Insert new coq cell above (${cmdOrCtrl}-Q)`, cmdInsertCoq(schema, filef, InsertionPlace.Above), false),
+        createMenuItem("Code ↓", "Insert new code cell underneath", cmdInsertCoq(schema, filef, InsertionPlace.Underneath), false),
+        createMenuItem("Code ↑", "Insert new code cell above", cmdInsertCoq(schema, filef, InsertionPlace.Above), false),
```
Comment (jim-portegies) [inline editor/src/kroqed-editor/menubar/menubar.ts:146]:
```suggestion
        createMenuItem("Math ↓", "Insert new verified math block underneath", cmdInsertCoq(schema, filef, InsertionPlace.Underneath), false),
        createMenuItem("Math ↑", "Insert new verified math block above", cmdInsertCoq(schema, filef, InsertionPlace.Above), false),
```

**Review by jim-portegies — CHANGES_REQUESTED** [review]
Autocomplete works nicely now! Just a suggestion for different text on Code block.

**Comment by jim-portegies** [comment]
@XyntaxCS I think the styling when the menu buttons cover multiple lines is also not so nice yet. It may help to have a bit of vertical space between the buttons when there are multiple lines, and to have a balanced number of buttons on the first and the second line.

**Comment by XyntaxCS** [comment]
Yeah I agree. I was planning on discussing this on Friday since it looks a bit odd now. I will look at your suggestions later and will try out some things.

**Review by jim-portegies — APPROVED** [review]

**Comment by jim-portegies** [comment]
I'll merge this now, we can open a new pull request for the button layout

**Pull request merged**